### PR TITLE
Proposed replacement for OsmAnd section

### PR DESCRIPTION
**Pull request by gitGOwin**
* all instructions and images from OsmAnd 2.x, with Material Design interface.
* significant changes in structure, text and images.
* new section: how use OsmAnd with FieldPapers and JOSM, as suggested by Nick Allen
* some instructions are potentially applicable to OsmAnd DVR, the the iOS platoform equivalent but needs to be verified (I don't use iOS)

### CODE REVIEW

**Comment by Nick-Tallguy** [comment]
Hi @gitGOwin ,

I've downloaded & run locally on my laptop. Looks pretty good, but there are a couple of bit that need resolving:

  +  Near the top, the section

----

<a href="https://play.google.com/store/apps/details?id=net.osmand">
  OsmAnd for Android
</a>

The iOS version is called for "OsmAnd DVR", and is available from iTunes.

<a href="https://itunes.apple.com/us/app/osmand-dvr/id963873905">
  OsmAnd DVR on iTunes
</a>

>Some of OsmAnd's Android platform features may not yet be available on the iOS edition. See the developers' [blog site](http://osmand.net/blog) for details.

----

does not render - looks like this
selection_083


  +   Images used are good, but need to be reduced in size so that the longest edge is no bigger than 520 pixels.  

  +  This will add a second section - we will have two sections on Osmand - suggest we run like this briefly while we check if all info is there. This new section is a very big section & I need to spend some time testing it out with my copy of OsmAND!

Thanks for your work on this - if you can update 1 & 2, that would be great. @althio is better at grammar etc than me and may have observations as well.

Thank again

Nick

**Comment by gitGOwin** [comment]
@Nick-Tallguy 
I've resized the files. How do I update this pull request?

re the Play store and iTunes badges. It seems to be rendering okay from my end. I couldn't find the issue.

File: _posts/en/mobile-mapping/0900-12-29-osmand2.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+title: OsmAnd 2.0+
+permalink: /en/mobile-mapping/osmand/
+lang: en
+category: mobile-mapping
+---
+
+OsmAnd
+======================================================
+
+OsmAnd (OSM Automated Navigation Directions) is an open source mapping, track recording, touring, and map viewing and editing application which utilizes the OpenStreetMap (OSM) database for both online and offline use.
```
Comment (althio) [inline _posts/en/mobile-mapping/0900-12-29-osmand2.md]: Is it really different?
> mapping vs. map viewing and editing

I would suggest:
... is an open source application for map viewing, editing, track recording and touring which ...

Also I may go as far as putting an emphasis on "**offline**" here, what do you think?

**Comment by althio** [comment]
OK, I am sorry :( I don't have the time for a full in-depth review here but that looks REALLY GOOD.

So... I say: we just ship!
What do you think @Nick-Tallguy @gitGOwin ?
We merge, get it online, review and get feedback?

**Comment by Nick-Tallguy** [comment]
Yes, merge. It looks good. There is always something that someone finds, but this looks good to me & the best testing platform is by end users. If needed we update later.

Good work @gitGOwin thank you.

**Comment by gitGOwin** [comment]
Thanks guys. I've found some minor edits but it doesn't affect the flow of thought or distract from the main idea so I'll leave that for the next update. :)

**Pull request merged**